time-series/time-series-api/src/main/java/com/powsybl/timeseries/TimeSeries.java

```diff
@@ -125,20 +131,32 @@ static StringTimeSeries createString(String name, TimeSeriesIndex index, String.
     }
 
     static Map<Integer, List<TimeSeries>> parseCsv(Path file) {
-        return parseCsv(file, TimeSeriesConstants.DEFAULT_SEPARATOR);
+        return parseCsv(file, TimeSeriesConstants.DEFAULT_SEPARATOR, true, TimeFormat.DATE_TIME);
+    }
+
+    static Map<Integer, List<TimeSeries>> parseCsv(Path file, boolean versioned, TimeFormat timeFormat) {
+        return parseCsv(file, TimeSeriesConstants.DEFAULT_SEPARATOR, versioned, timeFormat);
     }
 
     static Map<Integer, List<TimeSeries>> parseCsv(String csv, char separator) {
         try (BufferedReader reader = new BufferedReader(new StringReader(csv))) {
-            return parseCsv(reader, separator);
+            return parseCsv(reader, separator, true, TimeFormat.DATE_TIME);
         } catch (IOException e) {
             throw new UncheckedIOException(e);
         }
     }
 
-    static Map<Integer, List<TimeSeries>> parseCsv(Path file, char separator) {
+    static Map<Integer, List<TimeSeries>> parseCsv(String csv, char separator, boolean versioned, TimeFormat timeFormat) {
+        try (BufferedReader reader = new BufferedReader(new StringReader(csv))) {
+            return parseCsv(reader, separator, versioned, timeFormat);
+        } catch (IOException e) {
+            throw new UncheckedIOException(e);
+        }
+    }
+
+    static Map<Integer, List<TimeSeries>> parseCsv(Path file, char separator, boolean versioned, TimeFormat timeFormat) {
```

Comment:
This is too much parameters, add a `TimeSeries.CsvParserConfig`.

Reply:
Done